jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -209,11 +210,7 @@
 
               # GKE variables
               export CLUSTER_NAME=${{E2E_NAME}}
-              export ZONE="us-central1-f"
               export KUBE_GKE_NETWORK=${{E2E_NAME}}
-              export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/ci/staging"
```

Comment:
I think just unsetting this would be sufficient.

check_version_skew=false means we aren't testing the PR code at all, so what's the point?